test_stack.py

```diff
@@ -0,0 +1,52 @@
+from stack import StackItem
+from stack import StackFrame
+import pytest
+
+
+def test_item_data():
+    # Tests that "Bacon" is returned when calling .data on item
+    bacon = StackItem("Bacon")
+    assert bacon.data == "Bacon"
+
+
+def test_create_stack_frame():
+    # Tests that an empty stack can be created
+    new_stack = StackFrame()
+    assert new_stack
```

Comment:
this test is a little bit pointless.  there's essentially no way that this test fails if Python itself is working.